var/spack/repos/builtin/packages/py-carputils/package.py

```diff
@@ -25,16 +26,15 @@ class PyCarputils(PythonPackage):
     version("oc8.1", commit="a4210fcb0fe17226a1744ee9629f85b629decba3")
     version("oc7.0", commit="4c04db61744f2fb7665594d7c810699c5c55c77c")
 
-    depends_on("git")
+    depends_on("git", type=("build", "run"))
 
-    depends_on("python", type=("build", "run"))
-    depends_on("py-pip", type="build")
     depends_on("py-numpy@1.14.5:", type=("build", "run"))
-    depends_on("py-setuptools", type="build")
-    depends_on("py-python-dateutil", type="run")
-    depends_on("py-scipy@1.5.0:", type="run")
-    depends_on("py-matplotlib@3.0.0:", type="run")
-    depends_on("py-pandas", type="run")
-    depends_on("py-tables@3.6.1:", type="run")
-    depends_on("py-six@1.12.0:", type="run")
-    depends_on("py-ruamel-yaml", type="run")
+    depends_on("py-setuptools@41.6.0:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@41.6.0:", type=("build", "run"))
```
It's listed under `install_requires`